presto-main/src/main/java/io/prestosql/util/DisjointSet.java

```diff
@@ -29,7 +29,7 @@
         private T parent;
         private int rank; // Without path compression, this would be equal to depth. Depth of 1-node tree is considered 0.
 
-        public Entry()
```

Comment:
Technically, it doesn't matter since the class is private.
However, we soetimes use public methods in private classes to emphasize which methods are supposed to be used outside of the class itself and which shouldn't be (although this is not enforced by the compiler in any way).

If we maintain such style, then this change should be dropped.
If we don't want this style, then there are many more places that should be updated.

Reply:
Note that this change doesn't make it private, but just package private. We do use package private scope for nested classes methods and fields (e.g: `io.prestosql.sql.planner.iterative.rule.ReorderJoins`, but there are others).

Reply:
I know your preference. I also know some have different preference.  I think I'm OK either way.